src/client/rspamc.cxx

```diff
@@ -1003,7 +1079,7 @@ rspamc_symbols_output(FILE *out, ucl_object_t *obj)
 			emitted = (char *)ucl_object_emit(elt, UCL_EMIT_JSON);
 		}
 
-		fmt::print(out, "Urls: {}\n", emitted);
+		if (emitted && strcmp(emitted, "[]")) fmt::print(out, "Urls: {}\n", emitted);
```

Comment:
This will change the existing behaviour which is not a good thing. I'd suggest to use this logic merely for reports.

Reply:
I did this intentionally. But noted and will change it back. Thanks.